README.md

```diff
@@ -192,3 +193,23 @@ workflows:
                 - master
           run_group_id: "123"
 ```
+
+## Orb Release Process
+This section describes the release process for the orb itself:
+1. Create a feature branch and do your work
+1. Update the version in the repo: `git grep -z -l -F 1.1.0 | xargs -0 sed -i -e 's/1.1.0/1.2.0/g'`
+1. Push the feature branch to Github to kick off the
+   `lint-pack_validate_publish-dev` workflow in CircleCI. This may fail in
+   CircleCI with the error "Cannot find rainforest-qa/rainforest@dev:dev:alpha
+   in the orb registry". Fix this by running `circleci orb publish src/@orb.yml
+   rainforest-qa/rainforest@dev:alpha` and rerun the build
+1. When the `lint-pack_validate_publish-dev` workflow completes successfully,
+   it will trigger the `integration-tests_prod-release` workflow to test the
+   orb
+1. If the `integration-tests_prod-release` workflow passes, get review and
+   merge to master
+1. Tag master with the version: `git checkout master && git pull && git tag vX.Y.Z && git push origin vX.Y.Z`
```

Comment:
Can we ask that people use the `Github Releases` feature instead? It'll create the tag, but that way you're also prompted to put in a release name and changelog (since we don't have a `CHANGELOG.md`).

Reply:
Sure - can you update the PR with the documentation on how to do this and what the format of the messages in the release should be.

Reply:
Done 👍

Reply:
Cool - needs an approval now! @prayogoa  / @magni-